mindsdb/integrations/handlers/chromadb_handler/chromadb_handler.py

```diff
@@ -0,0 +1,368 @@
+from collections import OrderedDict
+from typing import Dict
+
+import chromadb
+import pandas as pd
+from chromadb import API
+from chromadb.config import Settings
+from integrations.handlers.chromadb_handler.settings import DEFAULT_EMBEDDINGS_MODEL
+from langchain.vectorstores import Chroma
+from mindsdb_sql import ASTNode, CreateTable, Insert, Select, Star
+
+from mindsdb.integrations.handlers.chromadb_handler.helpers import (
+    documents_to_df,
+    extract_collection_name,
+    get_metadata_filter,
+    load_embeddings_model,
+    split_documents,
+)
+from mindsdb.integrations.libs.base import VectorStoreHandler
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+from mindsdb.integrations.libs.response import RESPONSE_TYPE
+from mindsdb.integrations.libs.response import HandlerResponse as Response
+from mindsdb.integrations.libs.response import HandlerStatusResponse as StatusResponse
+from mindsdb.utilities import log
+
+# todo create separate util dir for vectorstore
```

Comment:
From today's discussion: we'll wait for the PgVector handler before refactoring anything out.